samza-core/src/test/java/org/apache/samza/task/TestAsyncRunLoop.java

```diff
@@ -567,15 +566,15 @@ public void testCommitBehaviourWhenAsyncCommitIsEnabled() throws InterruptedExce
         try {
           firstMsgCompletionLatch.await();
           secondMsgCompletionLatch.await();
-          Thread.sleep(100);
+          Thread.sleep(500);
```

Comment:
From what I understand, clock instance we pass in through AsyncRunloop constructor can't be used to control/achieve that.  Can you please throw more light on this comment?